testing/conversion/conversion_test.go

```diff
@@ -94,8 +95,11 @@ func dropDatabase(t *testing.T, dbPath string) {
 	}
 }
 
-func BuildConv(t *testing.T, numCols, numFks int) *internal.Conv {
+func BuildConv(t *testing.T, numCols, numFks int, makeEmpty bool) *internal.Conv {
 	conv := internal.MakeConv()
```

Comment:
nit; maybe just create another helper function as following (but feel free to ignore this comment if you think that would add more distraction):

func BuildEmptyConv() *internal.Conv {
       return internal.MakeConv()
}

Reply:
I put it in the same function instead of a separate one because when calling the BuildConv() from test cases, I dont need to make 2 flows, one for empty conv and one for non-empty conv.

Ex:
`for _, tc := range testCases {
		dbURI, err := conversion.CreateDatabase(projectID, instanceID, tc.dbName, BuildConv(t, 2, 0, tc.emptySchema), os.Stdout)
		`
With 2 separate functions, I need to add an if statement and do some branching here.